src/app/components/common-ui/WebForm/index.js

```diff
@@ -0,0 +1,335 @@
+import _ from 'underscore';
+import cn from 'classnames';
+import op from 'object-path';
+import PropTypes from 'prop-types';
+
+import React, {
+    forwardRef,
+    useEffect,
+    useLayoutEffect as useWindowEffect,
```

Comment:
Never seen this before

Reply:
This just makes useLayoutEffect SSR safe.